src/components/ReportActionItem/ReportActionItemImages.js

```diff
@@ -35,6 +35,15 @@ const defaultProps = {
     isHovered: false,
 };
 
+/**
+ * This component displays a row of images in a report action item like a card, such
+ * as report previews or money request previews which contain receipt images. The maximum of images
+ * shown in this row is dictated by the size prop, which, if not passed, is just the number of images.
+ * Otherwise, if size is passed and the number of images is over size, we show an small overlay on the
+ * last image of how many additional images there are. If passed, total prop can be used to change how this
+ * additional number when subtracted from size.
+ */
+
```

Comment:
```suggestion
```

Reply:
This extra newline is here so that we don't have an ESLint error for invalid JSDoc. Would you prefer an eslint-ignore? I think it's fine as-is

Reply:
Wat really, weird ESLint rule :D